addons/hw_drivers/controllers/driver.py

```diff
@@ -79,7 +80,19 @@ def download_logs(self):
         """
         Downloads the log file
         """
-        if tools.config['logfile']:
-            return http.Stream.from_path(tools.config['logfile']).get_response(
+        log_path = tools.config['logfile']
+        if log_path:
+            check = adler32(log_path.encode())
+            stat = os.stat(log_path)
+            # intentionally don't use Stream.from_path as the path used is not in the addons path
+            # for instance, for the iot-box it will be in /var/log/odoo
+            return http.Stream(
+                    type='path',
+                    path=log_path,
+                    download_name=os.path.basename(log_path),
+                    etag=f'{int(stat.st_mtime)}-{stat.st_size}-{check}',
```

Comment:
That's what werkzeug is doing inside its own `sendfile`. For consistency I would like we keep the same. Otherwise just the `stat.st_size` should be enough.